go-apps/meep-sandbox-ctrl/server/sandbox-ctrl.go

```diff
@@ -416,6 +416,339 @@ func ceGetActiveScenario(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprint(w, string(scenario))
 }
 
+// Retrieves the active scenario domains
+// GET /active/domains
+func ceGetActiveScenarioDomain(w http.ResponseWriter, r *http.Request) {
+	log.Debug("CEGetActiveScenarioDomain")
+
+	if sbxCtrl.activeModel == nil || !sbxCtrl.activeModel.Active {
+		http.Error(w, "No scenario is active", http.StatusNotFound)
+		return
+	}
+
+	//Retrieve query parameters and populate filter
+	var filter mod.NodeFindFilter
+	query := r.URL.Query()
+	minimizeStr := query.Get("minimize")
+	if minimizeStr == "true" {
+		filter.Minimize = true
+	} else {
+		filter.Minimize = false
+	}
+	childrenStr := query.Get("children")
+	if childrenStr == "true" {
+		filter.Children = true
+	} else {
+		filter.Children = false
+	}
+
+	filter.DomainName = query.Get("domain")
+	filter.DomainType = query.Get("domainType")
+	filter.ZoneName = query.Get("zone")
+	filter.NetworkLocationName = query.Get("networkLocation")
+	filter.NetworkLocationType = query.Get("networkLocationType")
+	filter.PhysicalLocationName = query.Get("physicalLocation")
+	filter.PhysicalLocationType = query.Get("physicalLocationType")
+	filter.ProcessName = query.Get("process")
+	filter.ProcessType = query.Get("processType")
+
+	log.Info("SIMON NEW")
```

Comment:
Remove this log and any others in the code.

Reply:
removed